main.js

```diff
@@ -0,0 +1,227 @@
+//Needs Player x Computer function
+//Needs a Timer
+//ReeStyle css to add the timer button
+
+//HTML Elements querySelector
+let gameCells = document.querySelectorAll(".game-Cell");
+let resetButton = document.querySelector(".reset");
+let startButton = document.querySelector(".start");
+let playerX = document.querySelector(".playerXplayer");
+let playerX1 = document.querySelector(".playerXcomputer");
+let statusDiv = document.querySelector(".status");
+let playX = document.querySelector(".playX");
+let playO = document.querySelector(".playO");
+
+//Variables
+let nextPlay = true;
+let gameStart = false;
+let timer;
+let interval = 0;
+let playMode = null;
+let win = null;
+let playerButton;
+let oponentButton;
+let clockStart;
+
+//Win conditionArray
+let winCondition = [
+  [0, 1, 2],
+  [3, 4, 5],
+  [6, 7, 8],
+  [0, 3, 6],
+  [1, 4, 7],
+  [2, 5, 8],
+  [0, 4, 8],
+  [2, 4, 6],
+];
+
+//----------------------------Check win function --------------------------------\\
+//needs to check if has a draw
+//Function to check each winCondition array has a x or o
+function checkWin() {
+  winCondition.forEach((combo) => {
+    if (document.getElementById(combo[0].toString()).innerHTML !== "") {
+      //Line 41 to 46 check each position
+      if (
+        document.getElementById(combo[0].toString()).innerHTML ===
+          document.getElementById(combo[1].toString()).innerHTML &&
+        document.getElementById(combo[0].toString()).innerHTML ===
+          document.getElementById(combo[2].toString()).innerHTML
+      ) {
+        if (document.getElementById(combo[2].toString()).innerHTML === "×") {
+          win = "x"; // sets the win to X
+          if (win === "x") {
+            alert("X  Wins!"); // Will display a alert
+            nextPlay = null; //This here will prevent the player to not place any value after the game has won
+            for (let cellDiv of gameCells) {
+              // This here will clear all the cellDivs
+              cellDiv.innerText = "";
+              cellDiv.innerText = "";
+            }
+          }
+        } else {
+          win = "o"; // Sets the win to O
+          if (win === "o") {
+            alert("O  Wins!"); // Will display a alert
+            nextPlay = null; //This here will prevent the player to not place any value after the game has won
+            for (let cellDiv of gameCells) {
+              //this will clear all the cellDivs
+              cellDiv.innerText = "";
+              cellDiv.innerText = "";
```

Comment:
you can add a draw feature by changing this if else statement around a little bit